### ISSUE TITLE
Create Client Error For Bundled JS

### ISSUE THREAD

**Issue by dev0x10** [braintree/braintree-web#219]
Braintree.client.create({
        authorization: 'token'
      }, (err, clientInstance) => {
    //client instance not created
})

I'm wondering why I always getting error on my bundled app. I'm using Webpack for bundling. On dev mode, there is no error and I can get the client instance. I'm sure the token is passed because I debug it.

``` json
{name: "BraintreeError", code: "CLIENT_INVALID_AUTHORIZATION", message: "Authorization is invalid. Make sure your client token or tokenization key is valid.", type: "MERCHANT", details: undefined}
```

**Comment by mrak** [comment]
Since this could be an issue with your authorization keys and potentially sensitive data, please contact our support team at support@braintreepayments.com to further help with your integration.

This could be related to our sandbox and production environments needing separate credentials.

**Comment by dev0x10** [comment]
I'm using Sandbox.
There is no problem with the credentials if the app not bundled. Maybe I will try by loading the Braintree lib from CDN instead of bundle it inside my app.

Thanks!

**Comment by cherijs** [comment]
@dev0x10 i have same problem - how you solved that?

**Comment by cherijs** [comment]
@dev0x10 hmm removed webpack.optimize.UglifyJsPlugin  from webpack production - now it's working.. i gues UglifyJsPlugin fked something up

**Comment by tehnomaag** [comment]
I hit the same issue with IE11 and went digging further with debugger.

Found out that the reason, at least in my case, was that IE's native `atob` function throws `Invalid calling object` errors, since this polyfill doesn't bind and call the global atob properly:

```var atobNormalized = typeof global.atob === 'function' ? global.atob : atob;```
https://github.com/braintree/braintree-web/blob/master/src/lib/polyfill.js#L3

Here's a screenshot of IE debugger paused in `createAuthorizationData` of `create-authorization-data.js` file, just calling `window.atob` will fail, while calling it with proper `this` set to `window` will succeed.

<img width="399" alt="ie-atob-bind" src="https://cloud.githubusercontent.com/assets/1394282/24609321/11e37d5e-1883-11e7-8201-b658fcc41e05.png">

Fix is either to use `atob.call(window, authorization)` in https://github.com/braintree/braintree-web/blob/master/src/lib/create-authorization-data.js#L43

Or in my case I solved it at webpack level by forcing braintree polyfill check always to fail and use their internal atob reimplementation:

```
        new webpack.DefinePlugin({
             // will fail and fall back to https://github.com/braintree/braintree-web/blob/master/src/lib/polyfill.js#L5
            "global.atob": "window._atobBugWorkaround",
        }),
```

**Comment by crookedneighbor** [comment]
@tehnomaag Thanks for pointing this out. I'm working on a fix now.

**Comment by crookedneighbor** [comment]
@tehnomaag The `atob` issue you pointed out was fixed in [3.12.0](https://github.com/braintree/braintree-web/blob/master/CHANGELOG.md#3120)
